core-aam/index.html

```diff
@@ -5711,6 +5711,48 @@ <h4>Not Mapped</h4>
               >. Its absence not only indicates that the accessible object is not grabbed, but further defines it as not grab-able. These cases are marked as "Not mapped" without an asterisk.
             </p>
           </section>
+          <h4 id="ariaActions"><code>aria-actions</code></h4>
+          <table aria-labelledby="ariaActions">
+            <tbody>
+              <tr>
+                <th>ARIA Specification</th>
+                <td>
+                  <a class="property-reference" href="#aria-actions"><code>aria-actions</code></a>
+                </td>
+              </tr>
+              <tr>
+                <th>MSAA + IAccessible2</th>
+                <td>
+                  <span class="property"
+                    >Interface: <code>IAccessibleAction</code>, using the accessible nodes matching `aria-actions` IDREFs, if the referenced objects are in the accessibility tree</span
+                  ><br />
+                  <span class="property">Object Attribute: <code>aria-actions</code> &lt;value&gt;</span>
```

Comment:
Why do we want to expose this? We don't normally expose idrefs in object attributes. IAccessibleAction should be sufficient.